docs/iot/customized-platform/customized-platform.md

```diff
@@ -0,0 +1,123 @@
+# Customized Platform Guide
+
+## Introduction
+
+Customized platform feature is introduced in Tizen IoT Preview 2, which helps you to create a customized platform image. It also helps you to selectively aggregate features and set of Tizen platform APIs. The customized platform image is created in Craftroom. For more information, see <https://craftroom.tizen.org/>.
+
+Using the customized platform image, you can:
+
+-   remove the unwanted modules.
+-   reduce the size of the image.
+
+<a name="howtomakea"></a>
+## Create Customized Platform Image
+
+This section explains how to create a customized platform image with an example. However, for more information on using Craftroom, see <https://craftroom.tizen.org/forum/>.
+
+To create a customized platform image:
+
+1.  Go to [craftroom.tizen.org](https://craftroom.tizen.org/forum/).
+2.  Log on using your Tizen account credentials.
+3.  Click **Workspace**.
+4.  Click **New**. The **New Craft** window appears.
+5.  Enter the required information and click **Create**.
+
+    The name of the newly created craft appears in the left navigation bar.
+
+6.  In the left navigation bar, go to **newly_created_craft_name** > **Craft Designer**.
+7.  Click **Create**.
+
+    ![Create Project](media/1_create.png)
+
+8.   Enter the required information in each of the following field:
+    1.  **Presets**: Predefined image settings. If you are using the ARTIK 530 or ARTIK 530S IoT module, then you can either select **IOT Headed ARTIK530 Image** or **IOT Headless ARTIK530 Image**. If you are using Raspberry Pi 3, then select **IOT Headless RPI3 Image**. The **IOT Headed Common Presets** specifies the preset of the Tizen Common profile. The **IOT Core Presets** specifies the preset of the minimum set of features.
+
+        ![Presets](media/a_presets.png)
+
+    2.  **BSP (H/W)**: The supported hardware abstraction layer (HAL) for the boards. If you are using the ARTIK 530 or ARTIK 530S IoT module, then you must select **ARTIK530 selection for HAL and Feature**. If you are using Raspberry Pi 3, then select **RPI3 selection for HAL and Feature**. Additionally, you have options to select other HAL and Features by clicking the plus icon adjacent to ARTIK 530 or RPI3. If you select ARTIK or RPI3 in **Presets**, then you do not have to select **Partition Presets**. The **Partition Presets** is used for IOT Headed Common Presets and IOT Core Presets only.
+
+        ![BSP (H/W)](media/b_BSP.png)
+
+    3.  **Feature**: Tizen API groups. To implement the feature in detail, you can add or remove the features depending on what you want to create. For more information on APIs and features, see [Basics of Tizen Native API Reference](https://developer.tizen.org/development/api-references/native-application).
```

Comment:
Just a question : Are we using a link to Tizen Developer Site instead of  `/native/api/overview.md` for native api here?

Reply:
done